### ISSUE TITLE
Enhancement: Use Single JSON Environment Variable for Chaos Experiment Configuration

### ISSUE THREAD

**Issue by andrestoll** [aws-cli-tools/chaos-lambda-extension#12]
Motivation:
Configuring the experiment through a single JSON environment variable would increase ease and readability.

Detailed Description:
I propose the introduction of a single environment variable, perhaps named CHAOS_CONFIG_JSON, that contains a JSON object with all the necessary configuration parameters for the chaos experiment. This approach would enable a more organized and readable configuration setup, especially for complex experiments with multiple settings.

Potential Challenges:
- Backward compatability
- Parsing overhead

**Comment by having-fun-serverless** [comment]
Sounds useful. My 2 cents:
* at the moment, do you see backward compatibility as something critical? I'm not sure too many users are using this extension, therefore the extra work for supporting it doesn't sound useful. 
* From a ui perspective , say you want to quickly change a parameter, it will be harder to do it through the aws console. The env editor in the Lambda console is horrible:-), so changes might require redeploy
* Maybe it's worth keeping key properties outside of the json, as a separate env variable, for example on / off switch.

**Comment by andrestoll** [comment]
True, the env variable UI is not user-friendly. But ideally one would change the parameters in the FIS experiment template (e.g. SSM document). I think it's a good idea to keep the on/off switch outside
